client/landing/stepper/declarative-flow/internals/steps-repository/design-choices/index.tsx

```diff
@@ -0,0 +1,93 @@
+import {
+	DEFAULT_ASSEMBLER_DESIGN,
```

Comment:
Should we be using `getAssemblerDesign()` instead of `DEFAULT_ASSEMBLER_DESIGN`?